docs/modules/enduser-setup.md

```diff
@@ -30,8 +65,8 @@ The following HTTP endpoints exist:
 |/generate_204|Returns a HTTP 204 status (expected by certain Android clients during Wi-Fi connectivity checks)|
 |/status|Returns plaintext status description, used by the web page|
 |/status.json|Returns a JSON payload containing the ESP8266's chip id in hexadecimal format and the status code: 0=Idle, 1=Connecting, 2=Wrong Password, 3=Network not Found, 4=Failed, 5=Success|
-|/setwifi|Endpoint intended for services to use for setting the wifi credentials. Identical to `/update` except returns the same payload as `/status.json` instead of redirecting to `/`.|
-|/update|Form submission target. Example: `http://example.com/update?wifi_ssid=foobar&wifi_password=CorrectHorseBatteryStaple`. Must be a GET request. Will redirect to `/` when complete. |
+|/setwifi|Endpoint intended for services to use for setting the wifi credentials. Identical to `/update` except must be used as a form POST and returns the same payload as `/status.json` instead of redirecting to `/`.|
+|/update|Form submission target. Example: `http://example.com/update?wifi_ssid=foobar&wifi_password=CorrectHorseBatteryStaple`. Must be a GET request. Will redirect to `/` when complete. Note: won't update the eus_params.lua file. |
```

Comment:
Then when is eus_params.lua being updated? I'm a bit puzzled?
Or is this meant for the /setwifi route?
It should be updated in both cases.

Reply:
The choice of HTTP API is somewhat of an acquired taste, I agree. However, I don't mind `/update` and `/setwifi` being distinctively different. I overhauled that table to make things a little clearer. See https://github.com/nodemcu/nodemcu-firmware/blob/e98a1042b05037c00e85151fcf8c28d4ea968b6b/docs/modules/enduser-setup.md#http-endpoints